src/components/Inputs/CustomComboBox.js

```diff
@@ -48,14 +46,14 @@ const CustomComboBox = ({
       size={size}
       options={store}
       key={value}
-      PopperComponent={PopperComponent}
+      PaperComponent={({ children }) => <Paper style={{ width: `${displayFieldWidth * 100}%` }}>{children}</Paper>}
       getOptionLabel={(option, value) => {
         if (typeof displayField == 'object') {
           const text = displayField
             .map(header => (option[header] ? option[header]?.toString() : header === '->' && header))
             ?.filter(item => item)
             ?.join(' ')
-          if (text) return text
+          if (text !== undefined) return text
```

Comment:
why?

Reply:
it show me undefined  in new mode data Grid